### DISCUSSION TITLE
displaying reranking score in the final result

### DISCUSSION THREAD

**Discussion by xinyuluo** [komoot/photon#685] [Q&A]
I am navigating Photon's code to see how queries are processed, but my lack of java knowledge is holding me back. I seem to be able to find a scoring function to rerank the result here: https://github.com/komoot/photon/blob/master/src/main/java/de/komoot/photon/elasticsearch/PhotonQueryBuilder.java#L126
Not fully sure if I'm at the right place. Also, is it possible to display the final scoring after reranking in the result?

**Comment by lonvia** [comment]
You can run queries with `debug=1` to get the final score.
